src/scorecard.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+class Scorecard {
+  constructor () {
+    this.frames = [];
+  }
+
+  isComplete() {
+    if(this.frames.length === 10) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  storeFrame(frame = new Frame) {
+    this.frames.push(frame);
+  }
+
+  isPreviousFrameAStrike(frame) {
+    if (this.frames.indexOf(frame) > 0) {
+      var i = this.frames.indexOf(frame);
+      var isAStrike = this.frames[i - 1].isStrike;
+      return isAStrike;
+    } else {
+      return false;
+    }
+  }
+
+  isPreviousFrameASpare(frame) {
+    if (this.frames.indexOf(frame) > 0) {
+      var i = this.frames.indexOf(frame);
```

Comment:
The logic for finding the previous frame is repeated in `isPreviousFrameAStrike` and this method. Could we extract this out to follow the DRY principle?